src/main/java/core/basesyntax/handler/ConsoleHandler.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.handler;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler implements Handler {
+    private BetDao betDao = new BetDaoImpl();
+
+    @Override
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+
+        while (true) {
+            Bet bet = null;
+            String input = scanner.nextLine();
+            if (input.equalsIgnoreCase("q")) {
+                return;
+            }
+            try {
+                String[] parseString = input.split(" ");
+                int value = Integer.parseInt(parseString[0]);
+                double risk = Integer.parseInt(parseString[1]);
+
+                checkNegativeValue(value);
+                checkNegativeRisk(risk);
```

Comment:
throwing and then catching RuntimeExceptions is not a good practice
let's remove private methods, combine the two checks in one `if` and sout some message, then call `continue`